src/tests/global-const-style.test.ts

```diff
@@ -169,6 +169,13 @@ ruleTesterTs.run('global-const-style', rule, {
       errors: [{ messageId: 'asConst' }],
       output: 'const COLORS: Colors = { primary: "#000", secondary: "#fff" } as const;',
     },
+    // Object with Record type annotation missing UPPER_SNAKE_CASE and as const
+    {
+      code: 'const displayableNotificationModes: Record<NotificationMode, string> = { sms: "SMS", email: "Email", push: "Push" };',
+      filename: 'test.ts',
+      errors: [{ messageId: 'asConst' }, { messageId: 'upperSnakeCase' }],
+      output: 'const DISPLAYABLE_NOTIFICATION_MODES: Record<NotificationMode, string> = { sms: "SMS", email: "Email", push: "Push" } as const;',
```

Comment:
```suggestion
      output: 'const DISPLAYABLE_NOTIFICATION_MODES: Record<NotificationMode, string> = { sms: "SMS", email: "Email", push: "Push" }',
```
This actually shouldn't have `as const` because it already has the `: Record<NotificationMode, string>`.